diff -urN Varkon_1.19D.orig/sources/EX/src/extcp.c Varkon_1.19D/sources/EX/src/extcp.c
--- Varkon_1.19D.orig/sources/EX/src/extcp.c	2010-05-29 18:46:34.000000000 +0200
+++ Varkon_1.19D/sources/EX/src/extcp.c	2010-05-29 18:47:40.000000000 +0200
@@ -237,7 +237,7 @@
    /* Copy the resolved information into the 'sockaddr_in' structure  */
    /*-----------------------------------------------------------------*/ 
    memset(&server_addr,0,sizeof(server_addr)); 
-   memcpy(&(server_addr.sin_addr),hp->h_addr,hp->h_length);
+   memcpy(&(server_addr.sin_addr),hp->h_addr_list[0],hp->h_length);
    server_addr.sin_family = hp->h_addrtype;
    server_addr.sin_port = htons((unsigned short)port);
                           /* Port must be in n.w.byte order */
